src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,14 +1,47 @@
-import React from 'react';
+import React, { useCallback } from 'react';
 import './MovieCard.scss';
 
+function hightlight(filter: string, str: string) {
+  if (!filter) {
+    return str;
+  }
+
+  const regexp = new RegExp(filter, 'ig');
+  const matchValue = str.match(regexp);
+
+  if (matchValue) {
+    return str.split(regexp).map((s, index, array) => {
+      if (index < array.length - 1) {
+        const c = matchValue.shift();
+
+        return (
+          <>
+            {s}
+            <span className="hightlight">{c}</span>
+          </>
+        );
+      }
+
+      return s;
+    });
```

Comment:
respect for making highlighting, really cool! Only one moment: better do not to use variables with one letter ('s' or 'c')